internal/benchrunner/runners/rally/runner.go

```diff
@@ -274,6 +341,43 @@ func (r *runner) setUp() error {
 	return nil
 }
 
+func (r *runner) extractSimulatedTemplate(indexTemplate string) (string, error) {
+	simulateTemplate, err := r.options.ESAPI.Indices.SimulateTemplate(r.options.ESAPI.Indices.SimulateTemplate.WithName(indexTemplate))
+	if err != nil {
+		return "", fmt.Errorf("error simulating template from composable template: %s: %w", indexTemplate, err)
+	}
+
+	templateBody, err := io.ReadAll(simulateTemplate.Body)
+	if err != nil {
+		return "", fmt.Errorf("error reading simulated template from composable template: %s: %w", indexTemplate, err)
+	}
+
+	var simulatedTemplate map[string]interface{}
+	err = json.Unmarshal(templateBody, &simulatedTemplate)
+	if err != nil {
+		return "", fmt.Errorf("error unmarshaling simulated template from composable template: %s: %w", indexTemplate, err)
+	}
+
+	simulatedTemplate["priority"] = 1000
+	simulatedTemplate["index_patterns"] = []string{indexTemplate + "-ep"}
+
+	indexTimeSeries := map[string]interface{}{
+		"start_time": "2000-01-01T00:00:00Z",
+		"end_time":   "2099-12-31T23:59:59Z",
```

Comment:
Could this be too broad? Could we calculate something more limited based on the current time?

Reply:
It is currently what is used by the ES team in rally tracks. Long term TSDB plans to remove this limitation so we can get rid of this hack again.